packages/core/stories/dialog/dialog.qa.stories.tsx

```diff
@@ -25,256 +30,220 @@ function FakeDialog({ children, status, id }: DialogProps) {
   return (
-    <QAContainer cols={3} height={300} itemPadding={3} width={3000} {...rest}>
-      {sizes.map((size) => {
-        return <DialogTemplate key={size} header={"Info Dialog"} size={size} />;
+    <StackLayout gap={1}>
+      {DensityValues.map((density) => {
+        return (
+          <Fragment key={density}>
+            <SaltProvider density={density}>
+              <DialogTemplate />
+            </SaltProvider>
+            <SaltProvider density={density} mode="dark">
+              <DialogTemplate />
+            </SaltProvider>
+            {VALIDATION_NAMED_STATUS.map((status) => (
+              <Fragment key={status}>
+                <SaltProvider density={density}>
+                  <DialogTemplate status={status} />
+                </SaltProvider>
+                <SaltProvider density={density} mode="dark">
+                  <DialogTemplate status={status} />
+                </SaltProvider>
+              </Fragment>
+            ))}
+          </Fragment>
+        );
       })}
-    </QAContainer>
-  );
-};
-
-SizeDialog.parameters = {
-  chromatic: { disableSnapshot: false },
-};
-
-export const InfoDialog: StoryFn<QAContainerProps> = (props) => {
-  const { ...rest } = props;
-  return (
-    <QAContainer cols={3} height={300} itemPadding={3} width={1300} {...rest}>
-      <DialogTemplate status={"info"} header={"Info Dialog"} />
-    </QAContainer>
-  );
-};
-
-InfoDialog.parameters = {
-  chromatic: { disableSnapshot: false },
-};
-
-export const SuccessDialog: StoryFn<QAContainerProps> = (props) => {
-  const { ...rest } = props;
-  return (
-    <QAContainer cols={3} height={300} itemPadding={3} width={1300} {...rest}>
-      <DialogTemplate status={"success"} header={"Success Dialog"} />
-    </QAContainer>
-  );
-};
-
-SuccessDialog.parameters = {
-  chromatic: { disableSnapshot: false },
-};
-
-export const WarningDialog: StoryFn<QAContainerProps> = (props) => {
-  const { ...rest } = props;
-  return (
-    <QAContainer cols={3} height={300} itemPadding={3} width={1300} {...rest}>
-      <DialogTemplate status={"warning"} header={"Warning Dialog"} />
-    </QAContainer>
+    </StackLayout>
   );
 };
 
-WarningDialog.parameters = {
-  chromatic: { disableSnapshot: false },
-};
-
-export const ErrorDialog: StoryFn<QAContainerProps> = (props) => {
-  const { ...rest } = props;
-  return (
-    <QAContainer cols={3} height={300} itemPadding={3} width={1300} {...rest}>
-      <DialogTemplate status={"error"} header={"Error Dialog"} />
-    </QAContainer>
+StatusVariants.parameters = {
+  chromatic: {
+    disableSnapshot: false,
+    modes: {
+      theme: {
+        themeNext: "disable",
+      },
+      themeNext: {
+        themeNext: "enable",
+        corner: "rounded",
+        accent: "teal",
+        // Ignore headingFont given font is not loaded
+      },
+    },
+  },
+};
+
+export const ContentVariants: StoryFn<QAContainerProps> = () => {
```

Comment:
Looking at Chromatic, there are some short content with blank spacing? Is it intended usage?

Reply:
yes, it is showing sticky footer :)